README.md

```diff
@@ -0,0 +1,137 @@
+
+    chromosome	start	end
+
+
+### CRAM support
+
+`bam-readcount` now supports CRAM files as input. 
+
+If a reference is specified with `-f`, it will override whatever is in
+the CRAM header.
+
+Otherwise, the reference(s) encoded in the CRAM header or a lookup by
+MD5 at ENA will be used.
+
+
+Output
+------
+
+Output is tab-separated with no header to `STDOUT`, one line per
+position:
+
+    chr	position	reference_base	depth	base:count:avg_mapping_quality:avg_basequality:avg_se_mapping_quality:num_plus_strand:num_minus_strand:avg_pos_as_fraction:avg_num_mismatches_as_fraction:avg_sum_mismatch_qualities:num_q2_containing_reads:avg_distance_to_q2_start_in_q2_reads:avg_clipped_length:avg_distance_to_effective_3p_end   ...
+
+There is one set of `:`-separated fields for each reported `base` with
+statistics on the set of reads containing that base:
+
+Field | Description
+----- | -----------
+base | The base, eg `C`
+count | Number of reads
+avg_mapping_quality | Mean mapping quality
+avg_basequality | Mean base quality
+avg_se_mapping_quality | Mean single ended mapping quality
+num_plus_strand | Number of reads on the plus/forward strand
+num_minus_strand | Number of reads on the minus/reverse strand
+avg_pos_as_fraction | Average position on the read as a fraction, calculated with respect to the length after clipping. This value is normalized to the center of the read: bases occurring strictly at the center of the read have a value of 1, those occurring strictly at the ends should approach a value of 0
+avg_num_mismatches_as_fraction | Average number of mismatches on these reads per base
+avg_sum_mismatch_qualities | Average sum of the base qualities of mismatches in the reads
+num_q2_containing_reads | Number of reads with q2 runs at the 3’ end
+avg_distance_to_q2_start_in_q2_reads | Average distance of position (as fraction of unclipped read length) to the start of the q2 run
+avg_clipped_length | Average clipped read length
+avg_distance_to_effective_3p_end | Average distance to the 3’ prime end of the read (as fraction of unclipped read length)
+
+
+### Per-library output
+
+With the `-p` option, each output line will have a set of `{}`-delimited
+results, one for each library:
+
+    chr	position	reference_base	depth	library_1_name	{	base:count:avg_mapping_quality:avg_basequality:avg_se_mapping_quality:num_plus_strand:num_minus_strand:avg_pos_as_fraction:avg_num_mismatches_as_fraction:avg_sum_mismatch_qualities:num_q2_containing_reads:avg_distance_to_q2_start_in_q2_reads:avg_clipped_length:avg_distance_to_effective_3p_end	}   ...   library_N_name	{	base:count:avg_mapping_quality:avg_basequality:avg_se_mapping_quality:num_plus_strand:num_minus_strand:avg_pos_as_fraction:avg_num_mismatches_as_fraction:avg_sum_mismatch_qualities:num_q2_containing_reads:avg_distance_to_q2_start_in_q2_reads:avg_clipped_length:avg_distance_to_effective_3p_end	}
+
+
+Bugs
+----
+
+### I get a segfault when attempting to run on a whole bam file
```

Comment:
Is this still a known bug with the new codebase? Is it actually a bug, or should it just be an FYI (or maybe even a check on the command string to return an error before crashing)?

Reply:
I'm not actually sure--thought I'd better leave it in just in case.